build.zig

```diff
@@ -229,6 +229,11 @@ pub fn build(b: *Builder) !void {
     const version = if (opt_version_string) |version| version else v: {
         const version_string = b.fmt("{d}.{d}.{d}", .{ zig_version.major, zig_version.minor, zig_version.patch });
 
+        // If we can't spawn a process for git, just trust the version string
```

Comment:
if we can't spawn a process for git, emit a fatal error here saying that it must be provided with `-Dversion-string`.